components/DetailsPage/index.js

```diff
@@ -0,0 +1,43 @@
+import Link from "next/link";
+import Image from "next/image";
+import React from "react";
+
+export default function Detail({ comic }) {
+  return (
+    <main>
+      <Link href="/add">zurück zur Übersicht</Link>
+      <div>
+        <>
```

Comment:
Why do you sorround this `Fragment` with a `<div>`?

Reply:
i removed it. thanks for this information